src/webview-src/components/App.tsx

```
@@ -571,6 +581,59 @@
     }
   }, [events.length]);
 
+  function handleEvent(e: unknown) {
+    // Be tolerant in tests: accept any object with a string 'type' field
```

Biome (2.1.2)

[error] Shouldn't redeclare 'handleEvent'. Consider to delete it or rename it.

'handleEvent' is defined here:


(lint/suspicious/noRedeclare)